frontend/public/components/monitoring.tsx

```diff
@@ -1051,44 +1051,91 @@ const MetricsDropdown = ({onChange}) => {
   />;
 };
 
-const getGraphColor = colorIndex => graphColors[colorIndex % graphColors.length];
+const SeriesIcon = ({colorIndex, isDisabled, onClick}) => <div
+  className={classNames('query-browser-metric__color', {'query-browser-metric__color--disabled': isDisabled})}
+  onClick={onClick}
```

Comment:
Not new, but this isn't accessible. Ideally this would be a button styled as we need with an `aria-label`.

Reply:
Addressed by https://github.com/openshift/console/pull/1838